### PR TITLE
Ports our syndie items costs

### PR DESCRIPTION
**Pull request by FluffySurvivor**
And a few tweaks:

emag cost reduced to 8 from 9

binary encryption key cost reduced to 4 from 5.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
what did you change all those things from? those aren't the tg prices, did you just copypaste in old yog prices?

**Comment by Bawhoppen** [comment]
and by that I mean, the prices you were changing from, you said you changed emag cost from 9 to 8, but but emags are only 6 TC on tg

**Comment by FluffySurvivor** [comment]
From yogs..

**Comment by Bawhoppen** [comment]
no you misunderstand

**Comment by FluffySurvivor** [comment]
I think it was made clearly that we will not keep everything tg has with this rebase.

**Comment by Bawhoppen** [comment]
thats not what I meant, where are you getting the 9 tc for an emag number from? my only guess is that is from your old yog prices

**Comment by X-TheDark** [comment]
You are, indeed, correct, emag currently costs 6 (based on /tg/) and is being changed to 8, the 9->8 is just a minor misunderstanding.

**Comment by FluffySurvivor** [comment]
Old yog: 9TC
TG :6 TC
This PR: 8 TC (Changed from 9 due to feedback on yog stating the nerf was too big) 

It's basically a port+ changes, if that clears things up a little.

**Comment by FluffySurvivor** [comment]
THANK GOD

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -583,23 +603,27 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 			The concussive effect from the explosion will knock the recipient out for a short period, and deafen \
 			them for longer. Beware, it has a chance to detonate your PDA."
 	item = /obj/item/weapon/cartridge/syndicate
+
+	cost = 4
+
 	exclude_modes = list(/datum/game_mode/traitor/double_agents)
 	cost = 6
```
Comment (oisin100) [inline code/modules/uplink/uplink_item.dm]: 2 cost variables 1 item?

Comment (FluffySurvivor) [inline code/modules/uplink/uplink_item.dm]: dunno it seemed to fix the conflict

**Pull request merged**